src/App.jsx

```diff
@@ -7,24 +7,40 @@ import RestaurantList from "./components/Main/RestaurantList.jsx"
 import Restaurants from "./data/Restaurant.js";
 import { useState } from "react";
 
-function App() { 
+function App() {
   const [ category, setCategory ] = useState("전체");
   const restaurantList = Restaurants();
 
   const filteredRestaurants = category === "전체" ? restaurantList : restaurantList.filter(
     (restaurant) => restaurant.category === category
   );
 
+  const [ clickRestaurantItem, setClickRestaurantItem ] = useState(null);
```

Comment:
관리 하는 상태의 이름이 click이라 뭔가 행동을 하는 함수 같이 느껴지는데
값 자체에 초점을 맞춰서 선택하는이 아닌 선택 된레스토랑 처럼 바꾸는 건 어떨까요?

```suggestion
  const [ clickedRestaurantItem, setClickedRestaurantItem ] = useState(null);
```
아니면 restaurant.name과 값을 비교하고 있으니 `clickedRestaurantName` 도 괜찮을 거 같아요!

Reply:
`clicked` 가 더 좋아 보이네요 반영하겠습니다😊